aiounifi/models/device.py

```diff
@@ -330,6 +330,19 @@ class TypedDeviceUplink(TypedDict):
     uplink_remote_port: int
 
 
+class TypedDeviceUpdtimeStats(TypedDict):
+    """Device uptime stats type definition."""
+
+    WAN: TypedDeviceUpdtimeStatsWan
```

Comment:
These (WAN) are hardcoded right? its not possible to rename them?

Reply:
I'm pretty new to Unifi. 
It tried to click around and find a place to rename and couldn't find any